### DISCUSSION TITLE
Ctrl Z或返回上一步操作

### DISCUSSION THREAD

**Discussion by JessicaHu03** [illacloud/illa-builder#1751] [General]
搭建page想往container里面拉component的时候很容易要么没按上d/k导致顶部的container滑到了页面下部，需要删掉component重新调整container位置，要么触发了其他的快捷键突然跳页面了，想返回上一步的时候无法直接ctrl z，如果是用的illa cloud应该可以存log返回吧？

**Comment by AruSeito** [comment]
您好，目前该功能已上线。
